src/tests/ftest/cart/util/cart_logtest.py

```diff
@@ -209,7 +209,7 @@ def subtract(self, val):
                "DER_NOTLEADER(-2008): 'Not service leader'")
 
 # Functions that are never reported as errors.
-IGNORED_FUNCTIONS = ('sched_watchdog_post', 'rdb_timerd')
+IGNORED_FUNCTIONS = ('sched_watchdog_post', 'rdb_timerd', 'cont_open')
```

Comment:
Why are we ignoring `cont_open` here instead of fixing the errors? And what are the errors we're ignoring?